include/ocpp/v201/charge_point.hpp

```diff
@@ -726,6 +735,8 @@ class ChargePoint : public ChargePointInterface, private ocpp::ChargingStationBa
 
     // Functional Block K: Smart Charging
     void handle_set_charging_profile_req(Call<SetChargingProfileRequest> call);
+    void handle_clear_charging_profile_req(Call<ClearChargingProfileRequest> call);
```

Comment:
Why do we pass these by value? We pass by value for all calls but I think it could very well be `const &`